Source/Entities/AHuman.cpp

```diff
@@ -2243,7 +2243,7 @@ void AHuman::PreControllerUpdate() {
 				} else if (heldDevice) {
 					if (HeldDevice* bgDevice = GetEquippedBGItem(); bgDevice && !heldDevice->IsOneHanded()) {
 						UnequipBGArm();
-					} else if (!bgDevice && !heldDevice->IsReloading() && heldDevice->IsSupportable()) {
+					} else if (!bgDevice && (!heldDevice->IsReloading() || (heldDevice->IsReloading() && heldDevice->GetUseSupportOffsetWhileReloading())) && heldDevice->IsSupportable()) {
```

Comment:
Checking `IsReloading()` twice is redundant.
```suggestion
					} else if (!bgDevice && (!heldDevice->IsReloading() || heldDevice->GetUseSupportOffsetWhileReloading()) && heldDevice->IsSupportable()) {
```